examples/promise-co-await/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015", "stage-2"],
```

Comment:
When using just a few features, it is best to use separate plugins. It keeps the compiled code a lot more like the original code.